src/imaginghelper.js

```diff
@@ -689,8 +689,17 @@
      **/
     function onOpen() {
         this._haveImage = true;
-        this.imgWidth = this._viewer.viewport.contentSize.x;
-        this.imgHeight = this._viewer.viewport.contentSize.y;
+        var contentSizeViewport = {},
+            contentSize;
+        // check if world is defined (OSD 2.2.0 and up)
+        if (this._viewer.world !== undefined){
```

Comment:
The world was actually introduced in 2.0.0, and contentSize was removed in 2.2.0, so checking on the world isn't a good test. How about testing the existence of contentSize instead?